module/exporter/migrations/Version20200122064958.php

```diff
@@ -40,7 +40,7 @@ public function up(Schema $schema): void
         $this->addSql(
             'ALTER TABLE exporter.export 
                     ADD CONSTRAINT export_channel_id_fk FOREIGN KEY (channel_id) 
-                    REFERENCES exporter.channel(id) ON UPDATE CASCADE ON DELETE RESTRICT'
+                    REFERENCES exporter.channel(id) ON UPDATE CASCADE ON DELETE CASCADE'
```

Comment:
You know that it will not work on current implementations ?

Reply:
this should changes be in new migration